independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/IDELauncherImpl.java

```diff
@@ -39,7 +40,8 @@ public static Closeable launch(Path projectRoot, Map<String, Object> context) {
                 Path launchingModulePath = QuarkusModelHelper.getClassPath(launchingModule);
                 // Gradle uses a different output directory for classes, we override the one used by the IDE
                 builder.setProjectRoot(launchingModulePath)
-                        .setApplicationRoot(launchingModulePath);
+                        .setApplicationRoot(launchingModulePath)
+                        .setTargetDirectory(launchingModulePath.getParent());
```

Comment:
I have no idea if this works in Gradle - I just copied what I did for Maven

Reply:
@glefloch do you know if this will work for Gradle?

Reply:
I just ran a quick test. 
The `launchingModulePath` points to `build/classes/java/main`. If you need to point to the module build dir, you can use `launchingModule.getBuildDir()` which points to `build/`or if you need the module root directory, you can use `launchingModule.getProjectRoot()`. So I think the should be `.setTargetDirectory(launchingModule.getProjectRoot())`

Reply:
Thanks @glefloch!